### PR TITLE
Automate the Config Creation Process

### PR DESCRIPTION
**Pull request by Abby-Wheelis**
After some trial and error, I have begun a workflow which will automatically create a PR for new config files after someone files the templated issue form to create them.

Steps:
1. deployer fills out a templated issue
2. creation/update of issue launches workflow that:
    - converts the issue to a json file
    - submits a PR to add the json file to our repo
3. TBD workflow for ensuring the PR is valid
4. We check the PR for validity and merge into repo
    - check that it is a valid deployment (not an attack)
    - check for associated "promised" files like onboarding survey, labels, or trip survey
    - may need to do some work by hand for some custom nitty gritty (still working out form templating for trip survey or custom reminder scheme cases)
    
This is in an early draft stage, as I still have things to work out:

- [x] make sure the workflow only goes when the templated issue is filed, not a normal issue
- [x] ask about config fields I was unsure about
- [ ] handle custom trip surveys
- [ ] handle custom reminder schemes
- [ ] set up validation (currently some degree by nature of the parsing process (won't work if things are missing), but I want to investigate testing it out)
- [x] write documentation for workflow and potential need to update the workflow

[Related Issue](https://github.com/e-mission/e-mission-docs/issues/1026)

### CODE REVIEW

**Comment by Abby-Wheelis** [comment]
More form updates as I practice using this in my local repo:

- [ ] set a default of false to trip end notifications
- [ ] set `autogen` and `token_generate` to both use the value of `autogen` (these need to match)

**Review by shankari — APPROVED** [review]
I'm going to merge this for now so that we can see how it looks, but have some comments that can be addressed in a future PR

**Comment by shankari** [comment]
Note that issues were not enabled on this repo, and I had to enable them to (hopefully) get this to work.

**Comment by shankari** [comment]
Seems to work now - I can at least create an issue...

**Comment by Abby-Wheelis** [comment]
For the actions to run, I think you'll need to make sure that workflows are enabled and that they have permission to create pull requests

File: .github/ISSUE_TEMPLATE/add-new-config.yml [hunk 1]
```diff
@@ -0,0 +1,552 @@
...
+    label: First Language
+    description: Select primary language.
+    options:
+      - en
+      - es
+      - lo
+  validations:
+    required: true
+- type: input
+  id: deployment_partner_name_lang1
```
[the remark this answers is not in the document]
Comment (Abby-Wheelis) [reply .github/ISSUE_TEMPLATE/add-new-config.yml:91]: I end up compiling it into an array, but this was the best way I found to format the template - the options are relatively limited, multiple templates could work

**Comment by shankari** [comment]
I am pretty sure that workflows are enabled.
wrt enabling creation of pull requests, I think you can use the `GitHub actions token" and enable pull requests on it
https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs

Do you want to create a test issue and see what it does?

**Comment by Abby-Wheelis** [comment]
It works! But is overzealous with the commits, I'll include a fix for that in my cleanup.

[see the PR here](https://github.com/e-mission/nrel-openpath-deploy-configs/pull/48#issuecomment-1817682417)

**Pull request merged**